core/container.go

```diff
@@ -66,6 +71,9 @@ type containerIDPayload struct {
 	// Image configuration (env, workdir, etc)
 	Config specs.ImageConfig `json:"cfg"`
 
+	// Pipeline
+	Pipeline PipelinePath `json:"group"`
```

Comment:
extreme nit: "pipeline" instead of "group"

Reply:
Thanks, yep, I missed a couple of those